### ISSUE TITLE
[Bug] TAG Website did not update with the latest blog

### ISSUE THREAD

**Issue by leonardpahlke** [cncf/tag-env-sustainability#331]
This PR https://github.com/cncf/tag-env-sustainability/pull/326 made changes to the [TAG ENV website](tag-env-sustainability.cncf.io). The deployment preview shows the new blog “Cloud Native Sustainability Week 2023 - putting sustainability on the community radar”. The changes were merged, all checks passed but the blog does not appear on the regular blog. Perhaps Netlify didn't picked up the change? 

@cjyabraham perhaps you could help with this. 

cc @guidemetothemoon

**Comment by leonardpahlke** [comment]
Thanks!!

**Comment by cjyabraham** [comment]
I've done a quick test following the same process as you have:
1. First I created a blog post from a fork with [this PR](https://github.com/cncf/tag-env-sustainability/pull/333). It showed up correctly on both the preview and live instances.
2. I then deleted the blog post with [this PR](https://github.com/cncf/tag-env-sustainability/pull/334). Again, both preview instance and live instance correctly reflected this change.

I did a bit of research on how netlify works with caches but couldn't come up with any theories on how to make this 100% reliable. We will, however, always have the ability to go in to netlify and manually trigger the build and reset the cache.

Maybe let's monitor this for now and see if this issue happens again?
